source/_data/blogs.yml

```diff
@@ -1,3 +1,28 @@
+- title: "Start your end to end testing with Angular and Cypress"
+  url: https://offering.solutions/blog/articles/2018/05/27/end-to-end-testing-with-angular-cypress/
+  sourceName: Offering Solutions Software
+  sourceUrl: https://offering.solutions/blog/
+  date: May 27, 2018
+  author: Fabian Gosebrink
+  img: /img/examples/e2e-with-angular-and-cypress.jgp
```

Comment:
Did you run this locally to ensure all of the images work correctly? I notice this has a `.jgp` extension, which does not look correct. You'll need to always run it locally and ensure the images and links are working.

Reply:
Good catch. Funny thing is, I _did_ and always do run it locally. And the incorrect file extension did work. 🤷🏻‍♀️
<img width="946" alt="screen shot 2018-06-05 at 10 31 44 am" src="https://user-images.githubusercontent.com/31670582/40982727-f5e29660-68ab-11e8-9bfb-ec2e2b675aa4.png">
<img width="679" alt="screen shot 2018-06-05 at 10 31 19 am" src="https://user-images.githubusercontent.com/31670582/40982732-f8773d2c-68ab-11e8-9605-66002ca7d997.png">

Will fix.

Reply:
Closer inspection reveals that the image extension is misspelled as well. So I guess that's why it worked. D'oh.

Reply:
🙃